apps/builder/app/@types/global.d.ts

```diff
@@ -0,0 +1,3 @@
+type Prettify<T> = { [k in keyof T]: T[k] } & {
```

Comment:
you sure you won't want this in packages as well? or in saas repo?

Reply:
Lets see if these guys are willing to add it https://github.com/sindresorhus/type-fest/issues/581